CONTRIBUTING.md

```diff
@@ -233,6 +233,10 @@ For a non-comprehensive but foundational overview of these best practices
 the [Effective Go](https://golang.org/doc/effective_go.html) documentation
 is an excellent starting place.
 
+We also recommend following the
+[Go Code Review Comments](https://go.dev/wiki/CodeReviewComments)
```

Comment:
> [The import . form can be useful in tests that, due to circular dependencies, cannot be made part of the package being tested](https://go.dev/wiki/CodeReviewComments#import-dot)

Don't we prefer to use `test` packages?

Reply:
> Don't we prefer to use dedicated test sub-packages?

I do not think that the guidelines are against our preference.
I think the most important part is that the Go team sees the import . as acceptable ONLY in this use case.

> Except for this one case, do not use import . in your programs.